dao/model.go

```diff
@@ -125,3 +125,12 @@ type BackupRequest struct {
 	SnapshotSpacePercent int
 	Excludes             []string
 }
+
+type BackupEstimate struct {
```

Comment:
Same comment as above - definitely a good idea, but would prefer to wait until post-Alderaan due to existing dependencies.